recipe/meta.yaml

```diff
@@ -1,16 +1,17 @@
-{% set version = "5.3.3" %}
+{% set version = "5.4.1" %}
```

Comment:
Is there a possibility to update to **5.4.2**?

Reply:
In that case we also need spyder-kernels 2.4.2. Other dependencies are already fulfilled.

Reply:
I will update to 5.4.2 after this update.